core/src/main/scala/dev/chronicles/core/VersionTracker.scala

```diff
@@ -128,13 +141,13 @@ object VersionTracker {
     * @param updates The list of updates that has been performed on the table.
     *                These must be returned in the order they were executed.
```

Comment:
Guess this no longer applies?

Reply:
Ah that definitely needs updating. The ordering of the stream is still important so probably worth specifying. Though it would probably be better expressed through a type, as you've suggested elsewhere, rather than comments that go out of date...